File: molecular/src/test/java/com/hartwig/actin/molecular/orange/evidence/curation/TestApplicabilityFilteringUtil.java
package com.hartwig.actin.molecular.orange.evidence.curation;

import org.jetbrains.annotations.NotNull;

public final class TestApplicabilityFilteringUtil {

    private TestApplicabilityFilteringUtil() {
    }

    @NotNull
    public static String nonApplicableGene() {
        return ApplicabilityFiltering.NON_APPLICABLE_GENES.iterator().next();
    }

    @NotNull
    public static String nonApplicableAmplification() {
        return ApplicabilityFiltering.NON_APPLICABLE_AMPLIFICATIONS.iterator().next();
    }
}

File: molecular/src/test/java/com/hartwig/actin/molecular/orange/evidence/curation/TestApplicabilityFilteringUtil.kt
package com.hartwig.actin.molecular.orange.evidence.curation

object TestApplicabilityFilteringUtil {

    fun nonApplicableGene(): String {
        return ApplicabilityFiltering.NON_APPLICABLE_GENES.iterator().next()
    }

    fun nonApplicableAmplification(): String {
        return ApplicabilityFiltering.NON_APPLICABLE_AMPLIFICATIONS.iterator().next()
    }
}
